docs/source/kedro_project_setup/configuration.md

```diff
@@ -255,7 +255,7 @@ Under the hood, `TemplatedConfigLoader` uses [`JMESPath` syntax](https://github.
 
 ### Jinja2 support
 
-From version 0.17.0, `TemplateConfigLoader` also supports the [Jinja2](https://palletsprojects.com/p/jinja/) template engine alongside the original template syntax. Below is an example of a `catalog.yml` file that uses both features:
+From version 0.17.0, `TemplatedConfigLoader` also supports the [Jinja2](https://palletsprojects.com/p/jinja/) template engine alongside the original template syntax. Below is an example of a `catalog.yml` file that uses both features:
```

Comment:
Sharing this because we're not using this feature more widely in our docs:

```suggestion
From version 0.17.0, {class}`~kedro.config.TemplatedConfigLoader` also supports the [Jinja2](https://palletsprojects.com/p/jinja/) template engine alongside the original template syntax. Below is an example of a `catalog.yml` file that uses both features:
```

This is clearly longer and more cumbersome to type, but generates a cross-reference to the `TemplatedConfigLoader` documentation, and emits a warning in case of a typo.

(I haven't tested this locally because emmmm I'm having some problems building the docs locally, working on it)

(In docstrings this is not needed, because these references are expanded automatically as long as they're wrapped in backquotes)

Reply:
These are all the objects in the intersphinx objects inventory that can be referenced from inside or outside the docs https://webknjaz.github.io/intersphinx-untangled/docs.kedro.org/

Reply:
The current state of our docs is... bad, so let's revisit this after I close https://github.com/kedro-org/kedro/issues/2454